catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1644,6 +1685,232 @@ public void setMetacardTypes(List<MetacardType> types) {
         this.metacardTypes = types;
     }
 
+    @Override
+    public CreateResponse create(CreateRequest createRequest) throws IngestException {
```

Comment:
This method should throw an UnsupportedOperationException if the CapabilitiesResponse does not include the "Transaction" operation, including support for the typeName we are attempting to publish.